src/controllers/check.your.answers.controller.ts

```diff
@@ -24,6 +23,9 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
       return next(e);
     }
   }
-
   return next(new Error("No Session present"));
 };
+
+export const post = (req: Request, res: Response) => {
+  res.redirect(OBJECTIONS_CONFIRMATION);
```

Comment:
add a return to this to prevent anything executing afterwards

Reply:
return added